### ISSUE TITLE
Questions about the training period.

### ISSUE THREAD

**Issue by Dennisasdzz** [XLearning-SCU/2022-TPAMI-DCP#4]
Hi, I have benefited greatly from your code, but I still have a small doubt：

In model.py - line 276, when the epoch reaches a certain stage (e.g. epoch>=100), the dual predictions loss function will only be involved in the gradient update.
However, this optimizer contains both auto-encoder and dual prediction parameters for training, so in the first 100 epochs, I observe that the dual prediction loss is still being computed (lines 260-265), will the parameters of the dual prediction  also be updated at this training period?

**Comment by Lin-Yijie** [comment]
Thank you for your interest! 
The dual prediction loss will not be backpropagated in the first 100 epochs. We compute it in the initial epochs to monitor its changes and the computational overhead for calculating this loss is negligible.

**Comment by Dennisasdzz** [comment]
Thank you for your prompt reply, I got it！
